src/util/drawing.js

```diff
@@ -90,14 +91,16 @@ export function path (context, options, fn) {
  * @param {Object} options
  * @param {Boolean} options.shadow - Whether to set any shadow options
  * @param {String} options.shadowColor - Default value: #000000
+ * @param {Number} options.shadowBlur - Default Value: 0
  * @param {Number} options.shadowOffsetX - Default value: 1
  * @param {Number} options.shadowOffsetY - Default value: 1
  */
 export function setShadow (context, options) {
   if (options.shadow) {
-    context.shadowColor = options.shadowColor || '#000000';
-    context.shadowOffsetX = options.shadowOffsetX || 1;
-    context.shadowOffsetY = options.shadowOffsetY || 1;
+    context.shadowColor = getDefault(options.shadowColor, '#000000');
+    context.shadowBlur = getDefault(options.shadowBlur, 0);
+    context.shadowOffsetX = getDefault(options.shadowOffsetX, 1);
+    context.shadowOffsetY = getDefault(options.shadowOffsetY, 1);
```

Comment:
This makes more sense. Wanting to be able to have an offset of 0 wouldn't be possible without this method.